doc/news/_preparation_next_release.md

```diff
@@ -181,6 +181,7 @@ plugins, bindings and tools are always up to date. Furthermore, we changed:
 
 We are currently working on following topics:
 
+- Merge tool in C _(Dominic Jäger)_
```

Comment:
This could be (much) more elaborate. Please also write in the tools section what you achieved so far.

Reply:
> Please also write in the tools section what you achieved so far.

As kdb is the tool with which we access cmerge, I assume that I write about the kdb integration alone in the tools section and about the rest in the outlook?

Reply:
It is also okay if you link to further documentation from the release notes.